generated/rflx-ipv4-generic_option.adb

```diff
@@ -442,7 +442,7 @@ is
                   Ctx.Cursors (Fld) := (State => S_Valid, First => Field_First (Ctx, Fld), Last => Field_Last (Ctx, Fld), Value => Value, Predecessor => Ctx.Cursors (Fld).Predecessor);
                end if;
                pragma Assert ((if Structural_Valid (Ctx.Cursors (F_Copied)) then
-                   (Ctx.Cursors (F_Copied).Last - Ctx.Cursors (F_Copied).First + 1) = IPv4.Flag_Base'Size
+                   (Ctx.Cursors (F_Copied).Last - Ctx.Cursors (F_Copied).First + 1) = Builtin_Types.Boolean_Base'Size
```

Comment:
Is this expected to be 2 or 8?

Reply:
Neither 2 nor 8. In a message a boolean field shall have a size of 1 bit.

Reply:
I meant 1 instead of 2.
In this case it might be helpful to add `with Size => 1` to https://github.com/Componolit/RecordFlux/blob/8dd61635ddaabec16afad279a6fcbdd09e236561/generated/rflx-builtin_types.ads#L17.

Reply:
After some discussion we decided to leave it as it is, as the size aspect does not make a difference in our case.